### PR TITLE
Add Messari crypto dd commands

### PR DESCRIPTION
**Pull request by jose-donato**
# Description

WIP

# How has this been tested?

* Please describe the tests that you ran to verify your changes. 
* Provide instructions so we can reproduce. 
* Please also list any relevant details for your test configuration.

# Checklist:

- [x] Update [our Hugo documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).
- [x] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [x] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [x] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/scripts).

# Others
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My code passes all the checks pylint, flake8, black, ... To speed up development you should run `pre-commit install`.
- [x] New and existing unit tests pass locally with my changes. You can test this locally using `pytest tests/...`.

### CODE REVIEW

File: tests/openbb_terminal/cryptocurrency/due_diligence/txt/test_dd_controller/test_print_help.txt [hunk 1]
```diff
@@ -37,3 +37,12 @@ Coinbase
    stats           show coin stats
 Messari
    mcapdom         show market cap dominance
+   mt              show messari timeseries
```
Comment (DidierRLopes) [inline tests/openbb_terminal/cryptocurrency/due_diligence/txt/test_dd_controller/test_print_help.txt:40]: no need for the "show" on the description in all of these

**Comment by minhhoang1023** [comment]
Excellent work 🚀 ✨ This totally revamped the DD menu

**Comment by minhhoang1023** [comment]
Was going through it and have a few comments:

**Roadmap command**
I remembered you mentioned the idea of plotting this as a plot, together with the price, to show how the news impacts the price. Shall we make it happen?

There's some href HTML tags left in the text
image

**Comment by minhhoang1023** [comment]
For circulating supply, what do you think about adding the price indicator? If the project keeps on minting new tokens without increasing demand, price will inevitably decrease as per econ 101.

image

**Comment by minhhoang1023** [comment]
For tk command, I got an error when loading ETH. Does it happen to you too?

image

Same with investors command
image

**Comment by minhhoang1023** [comment]
Already said it, but I'm going to say it again, you're killing this @jose-donato - basically adding half of what we are having to the dd menu 🚀 From my interviews with users, they absolutely need these commands. Instead of having to navigate through multiple sites, now they can find it all in the terminal.

**Comment by minhhoang1023** [comment]
A few small comments:

image

We should remove the list of time series in the `mt` command. The list is massive and i need to scroll a lot. Now that we have the query option, we no longer need this.

**Comment by minhhoang1023** [comment]
NaT should be displayed as Unknown / or Not specified - i'll leave it up to you.

image

File: openbb_terminal/cryptocurrency/due_diligence/messari_view.py [hunk 2]
```diff
@@ -48,33 +194,546 @@ def display_marketcap_dominance(
...
+    coingecko_symbol : str
+        Coingecko crypto symbol to check tokenomics
+    circ_supply_src : str
+        Source to look at circulating supply timeseries. Options: cg,mes
+    export : str
+        Export dataframe data to csv,json,xlsx file
+    external_axes : Optional[List[plt.Axes]], optional
+        External axes (1 axis is expected in the list), by default None
+    """
+    df, circ_df = get_tokenomics(coin, coingecko_symbol, circ_supply_src)
```
Comment (minhhoang1023) [inline openbb_terminal/cryptocurrency/due_diligence/messari_view.py:389]: Got this error msg if it's from cg

File: openbb_terminal/cryptocurrency/due_diligence/messari_view.py [hunk 3]
```diff
@@ -48,33 +194,546 @@ def display_marketcap_dominance(
...
+            "gov",
+            df,
+        )
+    else:
+        console.print(f"\n{coin} governance details not found\n")
+
+
+@log_start_end(log=logger)
+@check_api_key(["API_MESSARI_KEY"])
+def display_fundraising(
```
Comment (minhhoang1023) [inline openbb_terminal/cryptocurrency/due_diligence/messari_view.py:648]: Any idea why the investor % and org % doesn't match between the chart and text? With ETH it's matching tho so might be due to Messari itself?
image

**Comment by minhhoang1023** [comment]
I also added a prettify_paragraph to make the text look nicer :D

Before:
image

After:
image

image

**Comment by Chavithra** [comment]
@jose-donato Looks like the tests fail was due to file naming : case sensitivity

**Review by DidierRLopes — APPROVED** [review]

**Pull request merged**